Lib/multiprocessing/shared_memory.py

```diff
@@ -72,7 +72,7 @@ class SharedMemory:
     _mode = 0o600
     _prepend_leading_slash = True if _USE_POSIX else False
 
```

Comment:
I recommend adding `_track = True` here, since this appears to be a complete listing of all instance vars.